--- Do.Interface.Linux.Docky/src/Docky.Interface/TextRenderContext.cs	2009-06-28 18:19:29.000000000 -0600
+++ Do.Interface.Linux.Docky/src/Docky.Interface/TextRenderContext.cs	2010-09-01 13:11:53.000000000 -0600
@@ -45,12 +45,12 @@
 		
 		public TextRenderContext (Cairo.Context cr, string text, int width)
 		{
-			Context = cr;
-			Text = text;
-			Alignment = Alignment.Left;
-			MaximumWidth = width;
-			WrapMode = WrapMode.WordChar;
-			EllipsizeMode = EllipsizeMode.End;
+			this.Context = cr;
+			this.Text = text;
+			this.Alignment = Alignment.Left;
+			this.MaximumWidth = width;
+			this.WrapMode = WrapMode.WordChar;
+			this.EllipsizeMode = EllipsizeMode.End;
 		}
 	}
 }
