--- tools/js2c.py.orig	2010-07-25 22:20:45.000000000 +0200
+++ tools/js2c.py	2010-07-25 22:21:00.000000000 +0200
@@ -112,7 +112,7 @@
 def ExpandMacros(lines, macros):
   # We allow macros to depend on the previously declared macros, but
   # we don't allow self-dependecies or recursion.
-  for name_pattern, macro in reversed(macros):
+  for name_pattern, macro in reversed(list(macros)):
     pattern_match = name_pattern.search(lines, 0)
     while pattern_match is not None:
       # Scan over the arguments
