diff --git a/File/src/Do/Do.FilesAndFolders/IndexedFolder.cs b/File/src/Do/Do.FilesAndFolders/IndexedFolder.cs
index 33e2b41..1e3bd52 100644
--- a/File/src/Do/Do.FilesAndFolders/IndexedFolder.cs
+++ b/File/src/Do/Do.FilesAndFolders/IndexedFolder.cs
@@ -37,9 +37,9 @@ namespace Do.FilesAndFolders
 		{
 			if (path == null) throw new ArgumentNullException ("path");
 			
-			Path = Plugin.UnwrapHomeFolder (path);
-			Level = level;
-			Status = status;
+			this.Path = Plugin.UnwrapHomeFolder (path);
+			this.Level = level;
+			this.Status = status;
 		}
 		
 		public IndexedFolder (string path, uint level) : this (path, level, FolderStatus.Indexed)
