diff --git a/libexec/rbenv-init b/libexec/rbenv-init
index ed440d1..885e664 100755
--- a/libexec/rbenv-init
+++ b/libexec/rbenv-init
@@ -69,12 +69,6 @@ mkdir -p "${RBENV_ROOT}/"{shims,versions}
 
 echo 'export PATH="'${RBENV_ROOT}'/shims:${PATH}"'
 
-case "$shell" in
-bash | zsh )
-  echo "source \"$root/completions/rbenv.${shell}\""
-  ;;
-esac
-
 if [ -z "$no_rehash" ]; then
   echo 'rbenv rehash 2>/dev/null'
 fi
