--- launchy-vanilla/launchy-2.1.2/makefile	2008-07-26 19:55:39.000000000 -0400
+++ launchy-2.1.2/makefile	2008-10-02 19:52:46.000000000 -0400
@@ -4,7 +4,7 @@
 PLATFORMS_PATH=$(PREFIX)/lib/launchy/
 DESKTOP_PATH=$(PREFIX)/share/applications
 ICON_PATH=$(PREFIX)/share/pixmaps
-QMAKE=qmake-qt4
+QMAKE=qmake
 
 DEFS=SKINS_PATH=\\\"$(SKINS_PATH)\\\" PLUGINS_PATH=\\\"$(PLUGINS_PATH)\\\" \
 	PLATFORMS_PATH=\\\"$(PLATFORMS_PATH)\\\"
@@ -60,4 +60,4 @@
 	-rm -rf $(PLATFORMS_PATH)
 	-rm -rf $(SKINS_PATH)
 	-rm $(DESTDIR)$(DESKTOP_PATH)/launchy.desktop
-	-rm $(DESTDIR)$(ICON_PATH)/launchy.png	
\ No newline at end of file
+	-rm $(DESTDIR)$(ICON_PATH)/launchy.png	
