diff -rupN mongodb-src-r1.6.0/SConstruct mongodb-src-r1.6.0-r1/SConstruct
--- mongodb-src-r1.6.0/SConstruct	2010-08-05 15:32:06.000000000 +0200
+++ mongodb-src-r1.6.0-r1/SConstruct	2010-08-06 11:20:46.000000000 +0200
@@ -745,8 +745,6 @@ else:
 
 if nix:
     env.Append( CPPFLAGS="-fPIC -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch" )
-    if linux:
-        env.Append( CPPFLAGS=" -Werror " )
     env.Append( CXXFLAGS=" -Wnon-virtual-dtor " )
     env.Append( LINKFLAGS=" -fPIC -pthread -rdynamic" )
     env.Append( LIBS=[] )
