diff --git a/SConstruct b/SConstruct
index 5d0b835..3c6df12 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1043,9 +1043,11 @@ def ensureDir( name ):
             Exit( 1 )
 
 def ensureTestDirs():
-    ensureDir( "/tmp/unittest/" )
-    ensureDir( "/data/" )
-    ensureDir( "/data/db/" )
+    ensureDir( "testdir/" )
+    ensureDir( "testdir/tmp/" )
+    ensureDir( "testdir/tmp/unittest/" )
+    ensureDir( "testdir/data/" )
+    ensureDir( "testdir/data/db/" )
 
 def testSetup( env , target , source ):
     ensureTestDirs()
@@ -1120,7 +1122,7 @@ def startMongodForTests( env, target, source ):
     mongodForTestsPort = "40000"
     import os
     ensureTestDirs()
-    dirName = "/data/db/sconsTests/"
+    dirName = "testdir/data/db/sconsTests/"
     ensureDir( dirName )
     from subprocess import Popen
     mongodForTests = Popen( [ mongod[0].abspath, "--port", mongodForTestsPort, "--dbpath", dirName ] )
diff --git a/dbtests/dbtests.cpp b/dbtests/dbtests.cpp
index 3821163..983789a 100644
--- a/dbtests/dbtests.cpp
+++ b/dbtests/dbtests.cpp
@@ -22,6 +22,6 @@
 #include "dbtests.h"
 
 int main( int argc, char** argv ) {
-    return Suite::run(argc, argv, "/tmp/unittest");
+    return Suite::run(argc, argv, "testdir/tmp/unittest");
 }
 
