diff -ur mongodb-src-r1.4.0/SConstruct mongodb-src-r1.4.0.new/SConstruct
--- mongodb-src-r1.4.0/SConstruct	2010-03-02 07:37:07.000000000 -0800
+++ mongodb-src-r1.4.0.new/SConstruct	2010-03-07 22:40:02.000000000 -0800
@@ -1211,15 +1211,17 @@
             Exit( 1 )
 
 def ensureTestDirs():
-    ensureDir( "/tmp/unittest/" )
-    ensureDir( "/data/" )
-    ensureDir( "/data/db/" )
+    ensureDir( "testdir/" )
+    ensureDir( "testdir/tmp" )
+    ensureDir( "testdir/tmp/unittest/" )
+    ensureDir( "testdir/data/" )
+    ensureDir( "testdir/data/db/" )
 
 def testSetup( env , target , source ):
     ensureTestDirs()
 
 if len( COMMAND_LINE_TARGETS ) == 1 and str( COMMAND_LINE_TARGETS[0] ) == "test":
-    ensureDir( "/tmp/unittest/" );
+    ensureDir( "testdir/tmp/unittest/" );
 
 addSmoketest( "smoke", [ add_exe( "test" ) ] , [ test[ 0 ].abspath ] )
 addSmoketest( "smokePerf", [ "perftest" ] , [ perftest[ 0 ].abspath ] )
@@ -1297,7 +1299,7 @@
     mongodForTestsPort = "32000"
     import os
     ensureTestDirs()
-    dirName = "/data/db/sconsTests/"
+    dirName = "testdir/data/db/sconsTests/"
     ensureDir( dirName )
     from subprocess import Popen
     mongodForTests = Popen( [ mongod[0].abspath, "--port", mongodForTestsPort, "--dbpath", dirName ] )
diff -ur mongodb-src-r1.4.0/dbtests/dbtests.cpp mongodb-src-r1.4.0.new/dbtests/dbtests.cpp
--- mongodb-src-r1.4.0/dbtests/dbtests.cpp	2010-03-02 07:37:07.000000000 -0800
+++ mongodb-src-r1.4.0.new/dbtests/dbtests.cpp	2010-03-07 22:40:37.000000000 -0800
@@ -23,6 +23,6 @@
 
 int main( int argc, char** argv ) {
     static StaticObserver StaticObserver;
-    return Suite::run(argc, argv, "/tmp/unittest");
+    return Suite::run(argc, argv, "testdir/tmp/unittest");
 }
