diff -ur mongodb-src-r1.3.3/SConstruct mongodb-src-r1.3.3.new/SConstruct
--- mongodb-src-r1.3.3/SConstruct	2010-03-02 07:37:07.000000000 -0800
+++ mongodb-src-r1.3.3.new/SConstruct	2010-03-07 22:36:10.000000000 -0800
@@ -1478,8 +1478,6 @@
     fullInstallName = installDir + "/bin/" + name
 
     allBinaries += [ name ]
-    if (solaris or linux) and (not GetOption("nostrip")):
-        e.AddPostAction( inst, e.Action( 'strip ' + fullInstallName ) )
 
     if linux and len( COMMAND_LINE_TARGETS ) == 1 and str( COMMAND_LINE_TARGETS[0] ) == "s3dist":
         e.AddPostAction( inst , checkGlibc )
